### DISCUSSION TITLE
Using vercel edge function

### DISCUSSION THREAD

**Discussion by yuxxeun** [withastro/docs#3062] [Ideas]
i saw an [example](https://vercel.com/templates/astro/astro-edge-functions) implementation which using Astro and vercel edge fucntion, when im looking to the source, the code is pretty simple: just use `adapter: vercel()` and `output: server` in `astro.config.mjs`.

But, when i want to implementating to my site (which is using astro too), i face the error like this 
![gambar](https://user-images.githubusercontent.com/50067865/232374156-1b4e776b-113c-4cf1-b7db-a72a3d8a0788.png)

It seems, like `@astrojs/image` doesnt support for vercel adapter, anyone can give me insight about this problem?

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

I guess you fixed your issue by now. Not sure what was the solution in 2023 but it seems related to the default image service. Anyway, the discussions are intended to discuss improvements to the documentation; for any support questions, feel free to join our [Discord server](https://astro.build/chat)!

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
